data/json/recipes/weapon/mods.json

```diff
@@ -65,5 +65,98 @@
+    "time": 15000,
+    "decomp_learn": 2,
+    "book_learn": [ [ "manual_electronics", 2 ], [ "textbook_electronics", 2 ] ],
+    "using": [ [ "soldering_standard", 10 ], [ "surface_heat", 10 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "components": [ 
+        [ [ "plastic_chunk", 2 ] ],
+        [ [ "amplifier", 1 ] ],
+        [ [ "power_supply", 1 ] ],
+        [ [ "battery", 100 ], [ "plut_cell", 1 ] ],
+        [ [ "lens", 2 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "tele_sight",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "skills_required": [ [ "gun", 4 ] ],
+    "time": 15000,
+    "decomp_learn": 3,
+    "book_learn": [ [ "manual_rifle", 2 ], [ "manual_fabrication", 3 ], [ "textbook_fabrication", 3 ] ],
+    "using": [ [ "soldering_standard", 10 ], [ "surface_heat", 10 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "components": [ 
+        [ 
+            [ "scrap", 1 ], [ "can_drink_unsealed", 1 ], [ "can_food_unsealed", 1 ],
+            [ "canister_empty", 1 ], [ "plastic_chunk", 1 ]
+        ],
+        [ [ "lens", 2 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "pistol_scope",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "skills_required": [ [ "pistol", 5 ] ],
+    "time": 15000,
+    "decomp_learn": 4,
+    "book_learn": [ [ "manual_pistol", 2 ], [ "manual_fabrication", 3 ], [ "textbook_fabrication", 3 ] ],
+    "using": [ [ "soldering_standard", 10 ], [ "surface_heat", 10 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "components": [ 
+        [ 
+            [ "scrap", 1 ], [ "can_drink_unsealed", 1 ], [ "can_food_unsealed", 1 ],
+            [ "canister_empty", 1 ], [ "plastic_chunk", 1 ]
+        ],
+        [ [ "lens", 2 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "pistol_stock",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "skills_required": [ [ "pistol", 3 ] ],
+    "time": 15000,
+    "decomp_learn": 4,
+    "book_learn": [ [ "manual_pistol", 3 ], [ "manual_fabrication", 4 ], [ "textbook_fabrication", 4 ] ],
+    "using": [ [ "welding_standard", 15 ] ],
+    "qualities": [
+      { "id": "GLARE", "level": 2 },
+      { "id": "HAMMER_FINE", "level": 1 },
+      { "id": "SCREW_FINE", "level": 1 },
+      { "id": "SAW_M_FINE", "level": 1 }
+    ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "components": [ 
+        [ 
+            [ "scrap", 1 ], [ "can_drink_unsealed", 1 ], [ "can_food_unsealed", 1 ],
+            [ "canister_empty", 1 ], [ "plastic_chunk", 1 ]
+        ],
+        [ [ "lens", 2 ] ]
```

Comment:
Pistol stock crafted from lens?

Reply:
I seem to have pulled the components from the wrong part of my draft document. Fix incoming.